airbyte-config/persistence/src/test/java/io/airbyte/config/persistence/BaseDatabaseConfigPersistenceTest.java

```diff
@@ -56,6 +56,7 @@
 
   @BeforeAll
   public static void dbSetup() {
+    YamlSeedConfigPersistence.initialize(YamlSeedConfigPersistence.DEFAULT_SEED_DEFINITION_RESOURCE_CLASS);
```

Comment:
Can be we have some smiple helper method to construct this class, like `YamlSeedConfigPersistence.getDefault()`, which automatically use the default resource class? Otherwise people won't remember which class to use.

Reply:
This `initialize` method no longer exists. This line can be deleted.